python/ray/tests/test_metrics.py

```diff
@@ -1,12 +1,12 @@
 import os
 import json
 import grpc
-import psutil
 import pytest
 import requests
 import time
 
 import ray
+import psutil
```

Comment:
```suggestion
import psutil  # We must import psutil after ray because we bundle psutil with ray.
```

Also move this to after all of the Ray imports (separated by a new line).